person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  attr_writer :id
+  attr_accessor :name, :age
```

Comment:
- [ Optional] You did an amazing job to implement person class, but I would recommend to add both setters and getters on the top of the initialize method. it will make your code more clear  and  well organized  to read.